src/redux/actions/authActions.js

```diff
@@ -9,105 +9,122 @@ export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
 export const LOGOUT_REQUEST = 'LOGOUT_REQUEST';
 export const LOGOUT_FAILURE = 'LOGOUT_FAILURE';
 export const LOGOUT_SUCCESS = 'LOGOUT_SUCCESS';
-export const REQUEST_USER_INFO = 'REQUEST_USER_INFO';
-export const RECEIVE_USER_INFO_FAILURE = 'RECEIVE_USER_INFO_FAILURE';
-export const RECEIVE_USER_INFO_SUCCESS = 'RECEIVE_USER_INFO_SUCCESS';
+export const GET_USER_INFO_REQUEST = 'GET_USER_INFO_REQUEST';
+export const GET_USER_INFO_FAILURE = 'GET_USER_INFO_FAILURE';
+export const GET_USER_INFO_SUCCESS = 'GET_USER_INFO_SUCCESS';
 
 function requestSignup() {
   return {
     type: SIGNUP_REQUEST,
-    receivedAt: Date.now()
+    lastUpdated: Date.now(),
+    isFetching: true
```

Comment:
This looks great

Reply:
Awesome. I thought it would be a good idea explicitly declare the state changes in our Actions, and then simply use those changes in our Reducers using the Action object.